<?xml version="1.0" ?>
<launch>

  <arg name="robot_ip" default="192.168.110.2" />
  <arg name="load_gripper" default="true" />

  <include file="$(find franka_control)/launch/franka_control.launch">
    <arg name="robot_ip" value="$(arg robot_ip)" />
  </include>
  <include file="$(find franka_gripper)/launch/franka_gripper.launch">
    <arg name="robot_ip" value="$(arg robot_ip)" />
  </include>

  <!-- Set some project specific defaults for the panda arm. -->
  <node name="set_robot_defaults" pkg="franka_control_wrappers" type="set_panda_defaults.py" respawn="false" output="screen"/>

  <include file="$(find panda_moveit_config)/launch/panda_moveit.launch" />
  <include file="$(find panda_moveit_config)/launch/moveit_rviz.launch" />

  <include file="$(find franka_control_wrappers)/launch/cartesian_velocity_controller.launch">
    <arg name="stopped" value="true" />
  </include>

</launch>
